### PR TITLE
Added flake8 config's

### LINKED ISSUE #355: run flake8 on travis ci
- add Command to `.travis.yml`: `flake8 masonite/ --ignore=E501,F401,E128,E402,E731,F821,E712`
- add `flake8` to `requirements.txt`
- add `flake8` [configuration](http://flake8.pycqa.org/en/latest/user/configuration.html) to `setup.cfg`

### PR DESCRIPTION
**Pull request by w3x10e8**
This PR fixes #355

### CODE REVIEW

**Comment by vaibhavmule** [comment]
adding a note here, most of the pep8 error showing in Travis ci has been made as per pep8 on PR: #361

**Comment by w3x10e8** [comment]
@vaibhavmule will the errors will get fixed when #361 is merged?

**Comment by vaibhavmule** [comment]
Yes it will.

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,30 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description-file = README.md
+[flake8]
+ignore=E501,F401,E128,E402,E731,F821,E712
+exclude =
```
Comment (vaibhavmule) [inline setup.cfg]: just use include here, 

```
include=masonite, tests
```

That way only masonite folder will be targeted.

I'm not sure, if we should add `tests` folder for pep8 checking, @josephmancuso your thoughts?

**Pull request merged**